pennylane/transforms/invisible.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Contains a transform to make quantum function non-recordable
+or invisible within a QNode or quantum tape context."""
+
+from functools import wraps
+
+import pennylane as qml
+
+
+def invisible(fn):

```

Comment:
Not sure I would have picked `invisible`. My suggestions would be `no_recording` or `stop_recording`.

Reply:
Yeah, tbh this change was more about not being so keen on `no_recording` or `stop_recording` than it was about liking `invisible`.

My hesitancy is that we don't tend to use terminology such as 'recording' or 'queuing' at a high user-level, so this function might be a bit of an enigma.

Is there a better name we can use, that relates to building a function that is 'ignored' by a QNode?

Reply:
`no_apply`?

Actually I sort of like invisible now. Maybe we should keep it